loafwallet/Enter2FACodeView.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Enter2FACodeView.swift
+//  loafwallet
+//
+//  Created by Kerry Washington on 8/9/21.
+//  Copyright © 2021 Litecoin Foundation. All rights reserved.
+//
+
```

Comment:
Reusing this architecture for any modal view. The idea is presenting a customized modal view is a nice option as compared to Action Sheets or Alerts.  This view is presented in the View+Extension class.